# ultra_ranges
# Ranges in cm of the 10 ultrasonic distance sensors
int16 range_su0
int16 range_su1
int16 range_su2
int16 range_su3
int16 range_su4
int16 range_su5
int16 range_su6
int16 range_su7
int16 range_su8
int16 range_su9
